tests/test_formatter.py

```diff
@@ -203,3 +203,87 @@ def test_escape(self):
 
         # Escaping is atomic: brackets inside brackets are treated literally
         assert self.formatter.format(datetime(1, 1, 1), "[[[ ]]") == "[[ ]"
+
+
+class TestDateTimeFormatterPremadeFormats:
+    def setup_class(cls):
+        cls.formatter = formatter.DateTimeFormatter()
+
+    # cookie format: 'Thursday, 25-Dec-1975 14:15:16 EST'
+    def test_cookie(self):
+        tz = list(make_full_tz_list())[0]
```

Comment:
```suggestion
        tz = "America/New_York"
```
Since we only need one timezone let's avoid generating the whole list. This change can be replicated for each format.